getting_started_tutorials/cuml_sklearn_colab_demo.ipynb

```diff
@@ -0,0 +1,4404 @@
+{
```

Comment:
<p>They're not identical anymore. Would just cut the first part and minor tweak the rest.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>